ScienceDirect.js

```diff
@@ -573,7 +510,8 @@ var testCases = [
 					},
 					{
 						"title": "ScienceDirect Full Text PDF",
-						"mimeType": "application/pdf"
+						"mimeType": "application/pdf",
+						"proxy": false
```

Comment:
@dstillman What is this `proxy` field? Will it appear from now on always?

Reply:
Explained here:

https://www.zotero.org/support/dev/translators/coding#attachments

It's only necessary on a small number of sites.

Reply:
Ah, we actually add it some lines before. I was just wondering why this change occurred now within the testing in Scaffold. Maybe it is because of the new integration. nvm